trampoline/src/main/java/com/iluwatar/trampoline/Trampoline.java

```diff
@@ -98,7 +98,7 @@ public T get() {
         return trampoline(this);
       }
 
-      T trampoline(final Trampoline<T> trampoline) {
+      private T trampoline(final Trampoline<T> trampoline) {
```

Comment:
Could you please describe this change a bit, why is it needed? This particular snippet is also part of the pattern's `README.md` (which is rendered on the https://java-design-patterns.com web site) so all the changes should be reflected there.

Reply:
@charlesfinley can you please clarify on this part as why we modified this access specifier to `private`

We would love to merge the code to master.

Reply:
Will update soon with explanation. Sorry for the delay.

Reply:
@iluwatar @ohbus - Reverted the change. It's no longer required.